cmd/acra-server/acra-server.go

```diff
@@ -265,16 +267,21 @@ func main() {
 
 		ocspClientVerifier := network.DefaultOCSPVerifier{Config: *ocspClientConfig, Client: &network.DefaultOCSPClient{}}
 
-		crlConfig, err := network.NewCRLConfig(*tlsCrlUrl, *tlsCrlFromCert)
+		var crlClientConfig *network.CRLConfig
+		if len(*tlsCrlClientUrl) > 0 {
+			crlClientConfig, err = network.NewCRLConfig(*tlsCrlClientUrl, *tlsCrlFromCert)
+		} else {
+			crlClientConfig, err = network.NewCRLConfig(*tlsCrlUrl, *tlsCrlFromCert)
+		}
 		if err != nil {
 			log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorWrongConfiguration).
-				Errorln("Configuration error: invalid CRL config")
+				Errorf("Configuration error: invalid CRL config")
```

Comment:
why `Errorf` instead `Errorln` if we don't pass any param? If sometimes we back to the standard `log` package, it will print line without newline character)